addons/account_edi/views/account_move_views.xml

```diff
@@ -121,5 +121,16 @@
                 </xpath>
             </field>
         </record>
+        <record id="model_account_download_electronic_invoice" model="ir.actions.server">
+            <field name="name">Download Electronic Invoice</field>
```

Comment:
```suggestion
            <field name="name">Download Electronic Invoices</field>
```
I guess we could add a 's' since we are only using it in list view and the purpose is to download multiple at once.